.gitignore

```diff
@@ -19,6 +19,18 @@ src/tools/log-parser/kata-log-parser
 # Microsoft-specific
 .cargo/
 vendor/
+!src/tardev-snapshotter/vendor
```

Comment:
So we're INCLUDING `src/tardev-snapshotter/vendor` (and others) in source control - but I don't see them in this commit?

Reply:
![image](https://github.com/user-attachments/assets/779662f3-24d0-4c02-b69d-f0d64d615f91)
Not sure why its not showing here, probably because of large number of files. But cloning my branch will show you vendor folders under all the sub components i have mentioned. also shows up in git diff from terminal.